api/scanner/exif/exif_parser_external.go

```diff
@@ -196,6 +198,14 @@ func (p *externalExifParser) ParseExif(media_path string) (returnExif *models.Me
 		newExif.GPSLatitude = &latitudeRaw
 	}
 
+	// **Added GPS data validation**
+	// https://github.com/photoview/photoview/issues/850
+	if found_exif && (latitudeRaw < -90 || latitudeRaw > 90 || longitudeRaw < -90 || longitudeRaw > 90) {
```

Comment:
The below reads nicer and smoother the comments above this are not required. However, I personally I would at this stage extract the retrieval and validation of latitude and longitude into a function which returns the latitude and longtude to set on the data.

```go
func extractValidGpsData(fileInfo) (float, float) {
 // extract latitude and longitude, validate and return values.
}
```

```suggestion
	if found_exif && ( math.Abs(latitudeRaw) > 90 || math.Abs(longitudeRaw) > 90) {
```

As you've said before you're not a dev if you need more guidance feel free to reach out :)